src/apis/api.ts

```diff
@@ -52,6 +52,10 @@ export type FetchCallbacks<RawDataType> = (
   onError: (errorMsg: string) => void
 ) => () => void;
 
+// interface ConnectorWithNameSpace extends Connector {
+//   namespace_id: string;
+// }
```

Comment:
Worth deleting this, personally I never trust that commented out code in a repo is ever going to be helpful :smile:

Reply:
done